### ISSUE TITLE
[label-value-view] Accessibly Failures for Color Contrast 

### ISSUE THREAD

**Issue by cwalten** [cerner/terra-clinical#313]
# Issue Description
Our project recently moved to WebdriverI/O following the terra-toolkit upgrade. With that, we are wanting to use the beAccessible helper service, but it is failing on a Terra Clinical component.

## Issue Type

- [ ] New Feature
- [ ] Enhancement
- [ ] Bug
- [x] Other

## Expected Behavior
The Terra components should be able to pass the beAccessible service in terra-toolkit

## Current Behavior
Label Value View's label text has a contrast that is lower than the acceptable amount when placed on a default application background

**Comment by emilyrohrbough** [comment]
Color contrast issues were logged in https://github.com/cerner/terra-clinical/issues/306 and resolved in https://github.com/cerner/terra-clinical/pull/302/files. Looks like [tests need updated](https://github.com/cerner/terra-clinical/blob/master/packages/terra-clinical-label-value-view/tests/wdio/label-value-view-spec.js#L4) to account for this change.
